packages/product-feed-heureka/src/Form/HeurekaProductFormType.php

```diff
@@ -38,12 +40,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'currency' => 'CZK',
                 'scale' => 2,
                 'constraints' => [
-                    new Range([
-                        'min' => 0,
-                        'max' => 100,
+                    new MoneyRange([
+                        'min' => Money::fromInteger(1),
```

Comment:
There used to be `'min' => 0`. Is this change intentional or is it a mistake?

Reply:
I'd say it was a mistake
```suggestion
                        'min' => Money::fromInteger(0),
```

Reply:
I really don't know.. one more information - in other feeds it is `1`

Reply:
According to @TomasLudvik it was just a copy-paste mistake